celery/app/log.py

```diff
@@ -70,6 +72,9 @@ def setup(self, loglevel=None, logfile=None, redirect_stdouts=False,
             CELERY_LOG_LEVEL=str(loglevel) if loglevel else '',
             CELERY_LOG_FILE=str(logfile) if logfile else '',
         )
+        warnings.filterwarnings('always', category=CDeprecationWarning)
+        warnings.filterwarnings('always', category=CPendingDeprecationWarning)
+        logging.captureWarnings(True)
```

Comment:
I'm not 100% sure what this change is doing -- I gather it is somehow making it so that we show these warnings to users, but why isn't that happening anyway? Is it since warnings aren't shown unless you use `python -W`?

Reply:
Yes, that's exactly why.

Reply:
Neat. Thanks for confirming! I think this looks good then. 👍